### PR TITLE
URL maintainance: Add redirects and mark pages as gone 

### LINKED ISSUE #677: Remove old training pages from google index
A lot of our old training pages are still in Google's index and I think we should remove them, e.g.

* simplabs.com/training/2017-06-11-web-security
* simplabs.com/training/2017-02-16-sass-fundamentals

### PR DESCRIPTION
**Pull request by pichfl**
Closes #677 #684

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -12,318 +12,452 @@
...
+  from = "/ember/index.html"
+  to = "/expertise/ember/"
+
+[[redirects]]
+  from = "/blog/2015/12/03/ember-cli-deploy-notifications.html"
+  to = "/blog/2015/12/3/ember-cli-deploy-notifications/"
+
+[[redirects]]
+  from = "/blog/ember/2018/05/30/a-little-encouragement-goes-a-long-way-in-2018.html"
+  to = "/blog/2018/05/30/a-little-encouragement-goes-a-long-way-in-2018"
```
Comment (marcoow) [inline netlify.toml]: Can we move these next to the existing redirects for blog posts for better organization?

File: netlify.toml [hunk 2]
```diff
@@ -12,318 +12,452 @@
...
+  from = "/jobs"
+  to = "/"
+  status = 410
+  force = true
+
+[[redirects]]
+  from = "/jobs/*"
+  to = "/"
+  status = 410
+  force = true
```
Comment (marcoow) [inline netlify.toml]: Do we need the `force` here? If so we might need it for the other ones as well?

File: netlify.toml [hunk 3]
```diff
@@ -12,318 +12,452 @@
...
-  from="https://phoenix-workshops.simplabs.com"
-  to="https://simplabs.com/services/tutoring/"
+  from = "https://phoenix-workshops.simplabs.com"
+  to = "https://simplabs.com/services/tutoring/"
 
 [[redirects]]
-  from="https://phoenix-workshops.simplabs.com/*"
-  to="https://simplabs.com/services/tutoring/"
+  from = "https://phoenix-workshops.simplabs.com/*"
+  to = "https://simplabs.com/services/tutoring/"
```
Comment (marcoow) [inline netlify.toml:334]: I guess we should add status 410 for these redirects as well?

File: netlify.toml [hunk 4]
```diff
@@ -12,318 +12,452 @@
...
+  from = "training/2017-02-17-modern-seo"
+  to = "/"
+  status = 410
+  force = true
+
+[[redirects]]
+  from = "training/2017-02-17-modern-seo/*"
+  to = "/"
+  status = 410
+  force = true
```
Comment (marcoow) [inline netlify.toml]: Can we redirect these URLs to the tutoring service page (https://simplabs.com/services/tutoring/) as we do for the old workshop domains?

File: netlify.toml (hunk 3 above)
Comment (marcoow) [reply netlify.toml:334]: Hm, just read the docs for 410 and they say:

> The resource is no longer available at the requested URI and no redirection will be given.

so I assume a redirect with status 410 is wrong anyway?

File: netlify.toml (hunk 2 above)
Comment (pichfl) [reply netlify.toml]: You're right, after re-evaluating the docs from Netlify, we should be able to omit `force`.

File: netlify.toml (hunk 3 above)
Comment (pichfl) [reply netlify.toml:334]: Netlify offers this as the only way to generate 410 responses, which indeed is a bit weird. 
We should only use 410 for pages that no longer have matching content and I tried to map as much as possible.

File: netlify.toml [hunk 5]
```diff
@@ -12,318 +12,432 @@
...
+  from = "/blog/ember/2018/05/30/a-little-encouragement-goes-a-long-way-in-2018.html"
+  to = "/blog/2018/05/30/a-little-encouragement-goes-a-long-way-in-2018"
 
 # redirect old workshop pages
 
 [[redirects]]
-  from="https://elixir-phoenix-workshop.simplabs.com"
-  to="https://simplabs.com/services/tutoring/"
+  from = "https://elixir-phoenix-workshop.simplabs.com"
+  to = "/services/tutoring/"
```
Comment (marcoow) [inline netlify.toml]: Are you sure this is correct? I'd assume this does not change the domain and redirect to `https://elixir-phoenix-workshop.simplabs.com/services/tutoring/`.

File: netlify.toml [hunk 6]
```diff
@@ -12,318 +12,432 @@
...
+  from = "/training/2017-02-16-sass-fundamentals"
+  to = "/services/tutoring/"
+
+[[redirects]]
+  from = "/training/2017-02-16-sass-fundamentals/*"
+  to = "/services/tutoring/"
+
+[[redirects]]
+  from = "/training/2017-02-17-modern-seo.html"
+  to = "/services/tutoring/"
```
Comment (marcoow) [inline netlify.toml:362]: Should these not have status 410 as well then?

Comment (pichfl) [reply netlify.toml:362]: 410 should only be used if a content is gone without an appropriate replacement. We can consider the tutoring page a good replacement for the training pages as far as I'm concerned.

File: netlify.toml (hunk 5 above)
Comment (pichfl) [reply netlify.toml]: The documentation is rather inconclusive on this topic. I reverted just in case.

File: netlify.toml (hunk 6 above)
Comment (marcoow) [reply netlify.toml:362]: ok 👍

**Review by marcoow — APPROVED** [review]

**Pull request merged**